frontend/src/scenes/experiments/ExperimentForm.tsx

```diff
@@ -37,7 +38,25 @@ const ExperimentFormFields = (): JSX.Element => {
                     <LemonField
                         name="feature_flag_key"
                         label="Feature flag key"
-                        help="Each experiment is backed by a feature flag. You'll use this key in your code."
+                        help={
+                            <div className="flex items-center space-x-2">
+                                <span>
+                                    Each experiment is backed by a feature flag. You'll use this key in your&nbsp;code.
+                                </span>
+                                <LemonButton
+                                    type="secondary"
+                                    size="xsmall"
+                                    tooltip={
+                                        dynamicFeatureFlagKey
+                                            ? "Use '" + dynamicFeatureFlagKey + "' as the feature flag key."
+                                            : 'Fill out the experiment name first.'
+                                    }
+                                    onClick={setExperimentFeatureFlagKeyFromName}
```

Comment:
@jurajmajerik I figured out why the button didn't work!

Valid:

```tsx
<LemonField>
    <LemonInput>
</LemonField>
```

Invalid ([which I had originally](https://github.com/PostHog/posthog/pull/26074/commits/618e27efaad728252c8c56b1f806812f43f9500c#diff-b7954a0344c3a2cf5099bdc7466d1daf6d828f156b1640492cf4e5da420078baR39-R68)):

```tsx
<LemonField>
    <div>
        <LemonInput>
        <LemonButton>
    </div>
</LemonField>
```

Notably, it appears `<LemonField>` only accepts one child.